DependencyInjection/Configuration.php

```diff
@@ -73,6 +73,13 @@ public function getConfigTreeBuilder()
                             ->scalarNode('cache')->defaultValue('file')->cannotBeEmpty()->end()
                         ->end()
                     ->end()
+                    ->arrayNode('annotation_patterns')
+                        ->info('Allows you to configure additional annotation patterns. '
+                            .'Other bundles can automatically register patterns by calling JMSDiExtraBundle::addAnnotationPattern($pattern).')
+                        ->treatNullLike(array('JMS\DiExtraBundle\Annotation'))
```

Comment:
ok, I cannot do it with addDefaultsIfNotSet

Symfony\Component\Config\Definition\Exception\InvalidDefinitionException: ->addDefaultsIfNotSet() is not applicable to prototype nodes at path "jms_di_extra.annotation_patterns"